src/gromo/modules/conv2d_growing_module.py

```diff
@@ -120,6 +538,85 @@ def padding(self):
     def padding(self, value):
         self.layer.padding = value
 
+    @property
+    def stride(self) -> tuple[int, int]:
+        return self.layer.stride
+
+    def __out_dimension(self, dim: int) -> int:
+        return (
+            int(
+                (self.input_size[dim] - self.kernel_size[dim] + 2 * self.padding[dim])
+                / self.stride[dim]
+            )
+            + 1
+        )
```

Comment:
We could use `gromo.utils.tools.compute_output_shape_conv` to do the computation. (I you commit the suggestion it wont work as I did not put the import.)

```suggestion
        return compute_output_shape_conv(self.input_size, self.layer)
```

Reply:
I would prefer that there were an option on this function to skip all the checks

Reply:
I suggest we do this in a separate PR